### PR TITLE
feat: expose replaceHistory option via navigable tab component

### PR DESCRIPTION
**Pull request by arjunlalb**
## Description
expose replaceHistory option via navigable tab component

### CODE REVIEW

File: projects/components/src/tabs/navigable/navigable-tab-group.component.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export class NavigableTabGroupComponent implements AfterContentInit {
   });
 
   public onTabClick(tab: NavigableTabComponent): void {
-    this.navigationService.navigateWithinApp([tab.path], this.activatedRoute);
+    this.navigationService.navigateWithinApp([tab.path], this.activatedRoute, [], tab.replaceHistory);
```
Comment (anandtiwary) [inline projects/components/src/tabs/navigable/navigable-tab-group.component.ts:60]: did i accidentally forget to remove it? This function shouldn't be used. We need to add this option to the `buildNavigationParams`

Comment (aaron-steinfeld) [reply projects/components/src/tabs/navigable/navigable-tab-group.component.ts:60]: yep, this is nested inside an ht-link and should be deleted here and in the template

Comment (arjunlalb) [reply projects/components/src/tabs/navigable/navigable-tab-group.component.ts:60]: @anandtiwary fixed this. Used ht-link in the component.

**Review by aaron-steinfeld — APPROVED** [review]

File: projects/components/src/tabs/navigable/navigable-tab-group.component.ts [hunk 2]
```diff
@@ -14,17 +14,15 @@ import { NavigableTabComponent } from './navigable-tab.component';
       <nav mat-tab-nav-bar *htLetAsync="this.activeTab$ as activeTab" disableRipple>
         <ng-container *ngFor="let tab of this.tabs">
           <ng-container *ngIf="!tab.hidden">
-            <ht-link [paramsOrUrl]="buildNavigationParam | htMemoize: tab">
-              <div class="tab-button" *htIfFeature="tab.featureFlags | htFeature as featureState">
-                <a mat-tab-link (click)="this.onTabClick(tab)" class="tab-link" [active]="activeTab === tab">
```
Comment (anandtiwary) [inline projects/components/src/tabs/navigable/navigable-tab-group.component.ts:19]: Are you sure it is still able to set active attribute for mat-tab link? This is why I had kept the ht-link outside the mat-tab-link.

Comment (arjunlalb) [reply projects/components/src/tabs/navigable/navigable-tab-group.component.ts:19]: Yep. its setting active tab correctly.

Comment (anandtiwary) [reply projects/components/src/tabs/navigable/navigable-tab-group.component.ts:19]: Nice. Thank you!

**Pull request merged**